lighthouse-core/audits/ios-pwa-icon.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+
+/**
+ * @fileoverview Audits if a page's web app has a valid icon for iOS installability.
+ */
+
+class IosPwaIcon extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'ios-pwa-icon',
+      title: 'Web app has a valid iOS touch icon',
```

Comment:
Open for wording nits on `title`, `failureTitle`, and `description`.

Reply:
Seems fine to me! Though not sure if we should move away from "Web app" as the noun? Still see folks benefit if they dont consider themselves an app

I18n these?

Reply:
i18n'd! 🇮🇨